internal/db/schema/migrations/oss/postgres/34/02_vault_private_library.up.sql

```diff
@@ -0,0 +1,69 @@
+begin;
+
+-- Replaces view from 22/05_vault_private_library.up.sql
```

Comment:
Can you add a comment in `22/05_vault_private_library.up.sql` that the view is updated in `34/02_vault_private_library.up.sql`? (Example in [0/69_wh_session_facts.up.sql](https://github.com/hashicorp/boundary/blob/d4d81c24a6f79b3f5f12b86ce35fce2b1cab8208/internal/db/schema/migrations/oss/postgres/0/69_wh_session_facts.up.sql#L149))